INSTALL.md

```diff
@@ -46,7 +46,14 @@ There are a couple of environment variables you can set to configure OSEM.
 | OSEM_SMTP_PASSWORD		| *string*			| The password for the smtp server
 | OSEM_SMTP_AUTHENTICATION	| plain, login or cram_md5      | The auth method for the smtp server
 | OSEM_SMTP_DOMAIN		| opensuse.org			| The HELO domain for the smtp server
-| CLOUDINARY_URL		| *sting*			| Configure your cloudinary.com cloud name and api key/secret
+| CLOUDINARY_URL		| *string*			| Configure your cloudinary.com cloud name and api key/secret
+| STRIPE_PUBLISHABLE_KEY/STRIPE_SECRET_KEY    | *string*          | Publishable/Secret Keys for Stripe Gateway
+| OSEM_ICON    | *string*          | Sharable link for your organisation's icon
+
+### Online Ticket Payments
+We use [Stripe](https://stripe.com) for accepting your ticket payments securely over the web.
+Our application uses iFrame for accepting your user's payment details without storing them making the application PCI SAQ-A Compliant.
```

Comment:
Will organizers have to fill out the SAQ A questionnaire and send it to Stripe or something?